### ISSUE TITLE
Every time I update to the new version of veda, Atom asks to rebuild it because it has some dependencies on obsolete modules.

### ISSUE THREAD

**Issue by kerabromsmu** [fand/veda#38]

**Comment by fand** [comment]
Thx for report @kerabromsmu !

In fact I'm aware of this problem but turned a blind eye, because I don't know why it happens and it's not a critical ... 😅 
I have to face the problem now... 👀

**Comment by fand** [comment]
`chokidar` depends on `fsevents` and it's causing rebuild problem?
I'm Looking for chokider alternatives

**Comment by fand** [comment]
In v2.1.3 VEDA removed `chokidar` from its dependencies, but I still get 🐞 on the bottom of Atom because of `fsevents` and `uws` 😿 

```
Some installed packages could not be loaded because they contain native modules that were compiled for an earlier version of Atom.

veda 2.1.3

fsevents@1.1.3 – Cannot find module '/Users/amagitakayosi/src/github.com/fand/atom-veda/node_modules/fsevents/lib/binding/Release/electron-v1.6-darwin-x64/fse.node'
uws@0.14.5 – Compilation of µWebSockets has failed and there is no pre-compiled binary available for your system. Please install a supported C++11 compiler and reinstall the module 'uws'.
```

`uws` is installed as a dependency of `socket.io`, but `fsevents` seems to be only in devDependencies...?

**Comment by fand** [comment]
I found some devDependencies were installed as dependencies and they may caused rebuild.
Fixed them in v2.11.2.

I tested both on my MBP and windows PC, and the 🐞 doesn't appear now!!
I'll close this issue, and remove the paragraph from FAQ few days later.
